lib/std/simd.zig

```diff
@@ -0,0 +1,369 @@
+// Note: Some functions here are known to malfunction on MIPS.
+// Functions whose tests fail on MIPS will trigger compile errors when compiled for that platform.
```

Comment:
Super-brief introduction of SIMD and error behavior of SIMD belongs into the top level comment.

Reply:
Are you saying that you want me to write one? Thanks for the review, by the way.

Reply:
> Are you saying that you want me to write one?

Yup. It does not need to be perfect. Just summarizing what you are using.